example/ultravox_example/client.py

```diff
@@ -51,6 +54,18 @@ def on_error(error):
         logging.exception("Client error", exc_info=error)
         done.set()
 
+    def _get_secret_menu(params: dict[str, Any]) -> str:
+        result_dict = {
+            "date": datetime.date.today().isoformat(),
+            "specialItems": [
+                {"name": "Banana smoothie", "price": 3.99},
+                {"name": "Butter pecan ice cream (one scoop)", "price": 1.99},
+            ],
+        }
+        return json.dumps(result_dict)
+
+    session.register_tool_implementation("getSecretMenu", _get_secret_menu)
```

Comment:
Did you mean to leave this? My preference would be to just register for all tools and return a simple hardcoded response ("OK")

Reply:
Tools have to be registered by name.  I'm planning to add this to all the SDK examples to match flutter so I can test tool calls with Dr. Donut as part of my presubmit script (and so there's a real example that's easy to copy).